.travis.yml

```diff
@@ -1,14 +1,13 @@
 ---
-language: node_js
-node_js:
-  - "6"
+language: python
```

Comment:
no.
This is a node project, it must have node as its global language.
That is needed so that we can have multiple versions in the global matrix.

You need to create a coala stage which is python, as already requested at https://github.com/coala/git-task-list/pull/42#discussion_r194219588 , and you have been given the recipe even in https://github.com/coala/git-task-list/pull/40 and fixup https://github.com/coala/git-task-list/pull/55

Reply:
Was following your suggestion on gitter to follow `gci-leaders`. I thought you aware of my problem. And offering an easy suggestion.

I was using jobs and nodejs before and there was an error. Pushed before the hint is suggested.